src/runtime_src/core/pcie/linux/CMakeLists.txt

```diff
@@ -15,8 +15,8 @@ file(GLOB XRT_PCIE_LINUX_FILES
   "*.cpp"
   "../common/system_pcie.cpp"
   "../common/device_pcie.cpp"
-  "../../common/system.cpp"
-  "../../common/device.cpp"
+#  "../../common/system.cpp"
```

Comment:
Hmm...Is there a reason why these lines are simply not removed entirely instead of being commented out?